content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md

```diff
@@ -32,24 +32,213 @@ influxctl [flags] [command]
 
 | Command                                                                 | Description                            |
 | :---------------------------------------------------------------------- | :------------------------------------- |
```

Comment:
@sanderson @powersj Looking at `cli.go`, `influxctl login` no longer exists? Just curious - looks like we never documented it to begin with.

Reply:
Yeah, I think it only existed in one version and it never did get added.